src/bot/cogs/commands.py

```
@@ -148,6 +149,42 @@ async def execute_actions(self, message: discord.Message, cmd_data: dict, args:
+        
+        if name in ("reload", "load"):
+            if not args:
+                await message.add_reaction("❌")
+                return
+            try:
+                if name == "reload":
+                    await self.bot.reload_extension(f"cogs.{args[0]}")
+                else:
+                    await self.bot.load_extension(f"cogs.{args[0]}")
+                await message.add_reaction("✅")
+            except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)